src/pykeen/losses.py

```diff
@@ -319,6 +319,17 @@ def process_lcwa_scores(
         )
         return self(predictions, labels)
 
+    def get_config(self) -> MutableMapping[str, Any]:
+        """Return the configuration necessary to re-instantiate the module as a JSON-serializable dictionary."""
+        reduction_name = None
+        for key, value in _REDUCTION_METHODS.items():
+            if self._reduction_method is value:
```

Comment:
because pykeen.losses.Loss inherits from the PyTorch loss, there is always a `self.reduction` which has exactly the thing you want available